aether/sdk/conf/settings.py

```diff
@@ -559,12 +563,18 @@ def get_required(name):
         AWS_S3_REGION_NAME = get_required('AWS_S3_REGION_NAME')
         AWS_DEFAULT_ACL = get_required('AWS_DEFAULT_ACL')
 
+        if bool(os.environ.get('COLLECT_STATIC_FILES_ON_STORAGE')):
+            STATICFILES_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
+
     elif DJANGO_STORAGE_BACKEND == 'gcs':
```

Comment:
I was reading GCS docs and I didn't find any mention to the static files, it looks like it's only available for AWS and Azure, isn't it? :disappointed: 

Did you try it?

Reply:
Yes. I tried it, it works